src/network/networkd-can.c

```diff
@@ -227,3 +226,44 @@ int config_parse_can_control_mode(
         SET_FLAG(network->can_control_mode_flags, mask, r);
         return 0;
 }
+
+int config_parse_can_termination(
+                const char* unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Network *network = userdata;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        if (isempty(rvalue)) {
+                network->can_termination_set = false;
+                return 0;
+        }
+
+        r = parse_boolean(rvalue);
+        if (r >= 0)
+                network->can_termination = r ? CAN_TERMINATION_DEFAULT_OHM_VALUE : 0;
+        else {
+                r = safe_atou16(rvalue, &network->can_termination);
+                if (r < 0) {
+                        log_syntax(unit, LOG_WARNING, filename, line, r,
+                                   "Failed to parse CAN termination value, ignoring: %s", rvalue);
+                        return 0;
+                }
+        }
```

Comment:
so, the value "0" is parsed as boolean here, not as termination ohm value. This is intended? There's no ambiguity here?

Reply:
Ouch. Nice catch. "0" has fortunately no ambiguity, but "1" must be handled as 1 ohm instead of true.